src/openforms/conf/locale/nl/LC_MESSAGES/django.po

```diff
@@ -3658,9 +3657,9 @@ msgid ""
 "fill in the following code to retrieve the form submission:<br /><br /> "
 "%(code)s <br /><br />"
 msgstr ""
-"U wordt hierna omgeleid naar een inlogpagina. Nadat u ingelogd bent, kunt "
-"ude de volgende code invoeren om de formulierinzending op te halen:<br /><br"
-" /> %(code)s <br /><br />"
+"Daarna wordt u doorgestuurd naar een pagina waar u inlogt met uw DigiD. "
+"Nadat u bent ingelogd, voert u de volgende code in:<br /><br /> "
```

Comment:
looking at this diff, I have the impression you're editing the files directly instead of using rosetta in the admin, am I right?

Preferably we use Rosetta to get smaller diffs, since that's how the majority of translations are managed.

Reply:
Oh! Indeed i edited them manually :open_mouth: